concepticondata/conceptlists/Dellert-2017-1016.tsv

```diff
@@ -250,7 +250,7 @@ Dellert-2017-1016-248	248	339	-1.157994	Spiel	Spiel::N	game	игра	spielerisch
 Dellert-2017-1016-249	249	979	0.040639	Gesellschaft	Gesellschaft::N	company	общество	Begleitung	companionship	компания	COMPANIONSHIP	2983	
 Dellert-2017-1016-250	250	482	-1.120418	Hilfe	Hilfe::N	help	помощь				HELP (ASSISTANCE)	3467	!HELP (NOUN)
 Dellert-2017-1016-251	251	254	-1.173862	Freund	Freund::N	friend	друг	sehr nahestehender Mensch	person whose company one enjoys	связанный хорошими отношениями НАПР:друг детства	FRIEND	1325	
-Dellert-2017-1016-252	252	908	0.056385	Kamerad	Kamerad::N	companion	приятель	mit dem man Zeit verbringt	with whom one spends time	близкий знакомый			not distinct from FRIEND
+Dellert-2017-1016-252	252	908	0.056385	Kamerad	Kamerad::N	companion	приятель	mit dem man Zeit verbringt	with whom one spends time	близкий знакомый	COMPANION	3978	not distinct from FRIEND
```

Comment:
I stumbled over the last comment in the row: "not distinct from FRIEND". I think this means that it is the same concept.

Reply:
I think this comment means for their internal analysis, that it is very similar. The Russian term points to a rather loose friend. So it would be rather "Kumpel" in German. Kamerad in German itself is a bit no longer in use. But the abstract definition fits many usages of Kamerand and also Russian приятель, I'd say.